test/export_import_test.py

```diff
@@ -0,0 +1,122 @@
+        self.CREATE_PROFILES_ARGUMENTS = self.MICROSERVICE_OPTIONS.get("CREATE_PROFILES_ARGUMENTS", "")
+        self.CREATE_ASSET_HIERARCHY = self.MICROSERVICE_OPTIONS.get("CREATE_ASSET_HIERARCHY", "False")
+        self.DELETE_PROFILES = self.MICROSERVICE_OPTIONS.get("DELETE_PROFILES", "False")
+        self.device_model = {
+            "type": "Simulator",
+            "id": "sim_001_test",
+            "label": "Test Simulator #1",
+            "enabled": "true"
+        }
+
+    @patch('logging.Logger.error')
+    def test_export_import_profile_data(self, mock_error):
+        # Get current working directory
+        current_dir = os.getcwd()
+        # Change working directory to extras to run script and export data
+        try:
+            os.chdir("../simulators/extras") # IDE
+        except:
+            os.chdir("simulators/extras") # Command line
+
+        try:
+            external_device_id = "sim_001"
+            device_id = self.cumulocity_api.get_device_by_external_id(external_id=f"{external_device_id}")
+            profile_id = self.cumulocity_api.get_profile_id(deviceID=device_id)
+
+            log.info("Begin export data")
+            # Take export time period
+            date_from, date_to = Utils.set_time_period()
+            # Run the ExportProfileData.py script
+            call(["python", "ExportProfileData.py", "--device-ids", f"{profile_id}"])
+            filename = f"{external_device_id}_profile"
+            # Check if the sim_001_profile.json is created
+            profile_path = f"export_data/{filename}.json"
+            self.assertTrue(os.path.exists(profile_path), msg=f"{filename}.json not found")
+
+            # Open the JSON file and load its contents
+            data = load(profile_path)
+            # Check if the data file is empty
+            self.assertNotEqual(len(data),0, msg=f"No content in {filename}.json file")
+
+            log.info("Delete all extract data")
+            self.cumulocity_api.delete_alarms(date_from=date_from,date_to=date_to,device_id=profile_id)
+            self.cumulocity_api.delete_measurements(date_from=date_from,date_to=date_to,device_id=profile_id)
+
+            log.info("Begin import data")
+            # Run the ImportData.py script and get the exit code
+            exit_code = call(["python", "ImportData.py", "--ifiles", f"{filename}"])
+            # Check if the exit code is 0
+            self.assertEqual(exit_code, 0, msg="ImportData.py script failed to run")
+
+            log.info("Run the ExportProfileData.py script again to test the Import")
+            call(["python", "ExportProfileData.py", "--device-ids", f"{profile_id}"])
+            filename = f"{external_device_id}_profile"
+            # Check if the sim_001_profile.json is created
+            profile_path = f"export_data/{filename}.json"
+            self.assertTrue(os.path.exists(profile_path), msg=f"{filename}.json not found")
```

Comment:
- instead of running the export again, I'd prefer to check the alarms/measurements in the given time range of that device in C8Y directly
- if you want to run the export again, you should delete the json file first (it might still exist from the previous call) and you should extract the logic to check if the file exists and the file content into a separate method